### ISSUE TITLE
Bugs: Most Wanted List

### ISSUE THREAD

**Issue by gemfarmer** [18F/doi-extractives-data#984]
A list of current bugs. cc @meiqimichelle 

Im going to take the ones with my name next to them :wink:

CSS/layout:
* [x] #980 - **BRIAN** Responsive cards on homepage
* [ ] #949 - Site doesn't display properly on mobile
* [ ] #938 - Mobile menu gap ~~(was this the one with Eric's caching issue?)~~ nope, this one is real -- I think maybe related to saving a space for the glossary/nav scroll bar??
* [x] #850 - Diving graphic images replace _in review PR #987_ 
* [ ] #810 - Content hitting edge of browser
* [ ] #349 - font rendering bug
* [x] #875 - Lato too light on safari _DONE_
* [x] #954 - **Brian** Add curated search to nav
* [ ] #998 - **Brian** Replace large diving graphic images with latest (gray, no colors) ~~@gemfarmer is there an issue for this that you know about? @meiqimichelle is #988 what you are talking about?~~ will get images from @ericronne once he is done with charts/graphs

Enhancements CSS/layout
* [x] #956 - Explore Landing adjustments _in review PR #987_ 
* [x] #957 - How it works Landing adjustments _in review PR #987_ 

JS edits:
* [x] #944 **BRIAN** - tags in search results overrun on mobile *_PR #985_*
* [x] #928 **Brian** - 'active' state on narrative selectors (scroll and select)
* [x] #995 - **Brian** add subnav (similar to case studies) to long narrative pages 
* [x] #1007 - **Brian**  'active' state on case studies selector (scroll and select) *_PR #1008_*

Explore page-specific:
* [ ] #977 **BRIAN** - All Lands Production
* [x] #976 **BRIAN** - Explore map description displaying poorly *_PR #985_*
* [x] #892 **BRIAN** - Fix GDP unit selector *_PR #985_*
* [x] #986 **BRIAN** - toggle not switching back to show *_PR #985_*

**Comment by gemfarmer** [comment]
Closing and opening new issue with remaining todos as #1016
